src/Dapr.Client/Dapr.Client.csproj

```diff
@@ -8,5 +8,14 @@
   <PropertyGroup>
     <Description>This package contains the reference assemblies for developing services using Dapr.</Description>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
Dapr.Client.csproj targets netcoreapp3.1

Have you considered netstandard2.1?

Reply:
Thats the decision we made early in the project to sue .netcore 3.*. https://github.com/dapr/dotnet-sdk/issues/15

Reply:
Ok.

Normally I'd say a client library should target netstandard2.x to make it usable by the most people, but given how dapr is used, it is unlikely anyone will ever call a dapr service directly from a Xamarin/Unity/UWP app 😄

Reply:
Yeah we went through all of this already. Dapr is for server side apps, and it's a new-ish project so there's no legacy.